tests/consensus_tests/test_failed_snapshot_recovery.py

```
@@ -212,4 +213,10 @@ def test_failed_snapshot_recovery(tmp_path: pathlib.Path):
     new_dense_search_result = search(peer_api_uris[-1], dense_query_vector, query_city)
     assert len(new_dense_search_result) == len(initial_dense_search_result)
     for i in range(len(new_dense_search_result)):
-        assert new_dense_search_result[i]["id"] == initial_dense_search_result[i]["id"]
\ No newline at end of file
+        assert new_dense_search_result[i]["id"] == initial_dense_search_result[i]["id"]
+
+def test_new_fail():
+    assert False
```

Ruff (0.8.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)